lib/errors.ts

```diff
@@ -247,6 +247,24 @@ export class BalenaAmbiguousApplication extends BalenaError {
 }
 BalenaAmbiguousApplication.prototype.code = 'BalenaAmbiguousApplication';
 
+/**
+ * @summary Balena ambiguous release
+ * @class
+ * @public
+ *
+ * @param {(String)} release - release hash
+ * @return {Error} error instance
+ *
+ * @example
+ * throw new errors.BalenaAmbiguousRelease('7cf02a6')
+ */
+export class BalenaAmbiguousRelease extends BalenaError {
+	constructor(public release: string) {
+		super(`Release is ambiguous: ${release}`);
```

Comment:
The message is a bit "ambiguous" 😉. How about making it a bit more descriptive, such as 
```suggestion
		super(`Release is ambiguous: ${release}. There are multiple releases with the same commit ID`);
```

It would also be nice to not have the error at all and give a list of options to the user, just my personal opinion.

Reply:
I thought about it, but that wasn't consistent with the errors that we already have.
Let me push your suggestion but use "release hash" instead of "commit ID", since that's more relevant in the balena push/deploy era.

Reply:
@sradevski decided to use `There are multiple releases matching the release hash '${release}'` just to cover the case that it's a short uuid and we do prefix matching.

Reply:
> It would also be nice to not have the error at all and give a list of options to the user, just my personal opinion.

Giving a list of options to the user assumes that the user is using the "tool" (CLI or whatever else is using this module) in an interactive session, but the "tool" could be used inside scripts or in a CI environment. See: https://github.com/balena-io/balena-cli/issues/1089